Code/Tools/AssetProcessor/native/AssetManager/PathDependencyManager.cpp

```diff
@@ -414,6 +415,21 @@ namespace AssetProcessor
                 dependencyVector);
         }
 
+        AzToolsFramework::AssetDatabase::ProductDependencyDatabaseEntryContainer existingDependencies;
+        if (!m_stateData->GetDirectReverseProductDependenciesBySourceGuidAllPlatforms(sourceEntry.m_sourceGuid, existingDependencies))
+        {
+            AZ_Error("PathDependencyManager", false, "Failed to query existing product dependencies for source %s", sourceEntry.m_sourceName.c_str());
```

Comment:
It can be helpful to printout both the source name and the UUID, in case one is missing information.